scripts/extras/foglamp.sudoers_rh

```diff
@@ -1 +1 @@
-%sudo ALL=(ALL) NOPASSWD: /usr/bin/yum -y update, /usr/bin/yum -y install foglamp, /usr/bin/yum -y install /usr/local/foglamp/data/plugins/foglamp*.rpm, /usr/bin/yum list, /usr/bin/yum -y install foglamp*
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/yum update, /usr/bin/yum -y install foglamp, /usr/bin/yum -y install /usr/local/foglamp/data/plugins/foglamp*.rpm, /usr/bin/yum list available foglamp-*, /usr/bin/yum -y install foglamp*
```

Comment:
we need "-y" here. See FOGL-3256

Reply:
The we should change update.py to pass the -y flag as well